src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -63,6 +67,21 @@ if (window.__initialData) {
   ConfigStore.loadInitialData(window.__initialData);
 }
 
+// APM -------------------------------------------------------------
+const config = ConfigStore.getConfig();
+// This is just a simple gatekeeper to not enable apm for whole sentry.io at first
```

Comment:
Can we feature flag this instead?

Reply:
We could, but this anyway would just report transaction events to our account.
Enabling this for other orgs would just increase the data which we, in the beginning, don't want to.